addons/mail/models/discuss/discuss_channel.py

```diff
@@ -304,7 +304,7 @@ def create(self, vals_list):
                 if cmd[0] != 0:
                     raise ValidationError(_('Invalid value when creating a channel with memberships, only 0 is allowed.'))
                 for field_name in cmd[2]:
-                    if field_name not in ["partner_id", "guest_id", "unpin_dt", "last_interest_dt"]:
+                    if field_name not in ["member_type", "partner_id", "guest_id", "unpin_dt", "last_interest_dt"]:
```

Comment:
Technically it would be best if it wasn't possible to pass `member_type` through create API calls.